src/main/java/com/piekoszek/nowaksharedrent/auth/InMemoryAccountRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.piekoszek.nowaksharedrent.auth;
+
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+
+class InMemoryAccountRepository implements AccountRepository {
+    private ConcurrentHashMap<String, Account> inMemoryAccountRepository = new ConcurrentHashMap();
```

Comment:
As field type we should use Interfaces if possible, so it should be
 `private Map<String, Account> inMemoryAccountRepository = new ConcurrentHashMap<>();`